doc/man3/SSL_set_connect_state.pod

```diff
@@ -39,11 +44,19 @@ using L<SSL_write_ex(3)>, L<SSL_write(3)>, L<SSL_read_ex(3)>, or L<SSL_read(3)>,
 the handshake routines must be explicitly set in advance using either
 SSL_set_connect_state() or SSL_set_accept_state().
 
+If SSL_is_server() is called before SSL_set_connect_state() or
+SSL_set_accept_state() is called (either automatically or explicitly),
+the result depends on what method is used when SSL_CTX is created with
+L<SSL_CTX_new(3)>. If a generic method or a dedicated server method is
```

Comment:
I think we want to use the past tense for a lot of this, something like:

"... the result depends on what method was used to instantiate the B<SSL_CTX> that was used to generate B<ssl>."
"... was passed to SSL_CTX_new(), ..."